README.md

```diff
@@ -94,3 +94,7 @@ For a more detailed and step-by-step introduction to Kubeapps, read our [introdu
 * [Detailed installation instructions](chart/kubeapps/README.md)
 * [Kubeapps Dashboard documentation](docs/user/dashboard.md)
 * [Kubeapps components](docs/architecture/overview.md)
+
+## Community
+
+* [#kubeapps on Kubernetes Slack](https://kubernetes.slack.com/messages/C9D3TSUG4)
```

Comment:
Helm links to `https://kubernetes.slack.com/messages/helm-users` so we should be able to use `https://kubernetes.slack.com/messages/kubeapps`. Also we should instruct people on how to join this slack (link to slack.k8s.io, as it's not obvious.

Reply:
+1

Reply:
> Helm links to https://kubernetes.slack.com/messages/helm-users so we should be able to use https://kubernetes.slack.com/messages/kubeapps.

The reason why I used the ID is because it seems that sometimes using `/kubeapps` the link gets lost after signing in and was landing in #announcements (I reproduced it a couple of times, but is not constant). I'll change it though.

Thanks!